java/configuration/src/main/java/sleeper/configuration/properties/instance/PartitionSplittingProperty.java

```diff
@@ -33,6 +34,14 @@ public interface PartitionSplittingProperty {
                     "maximum number of files that are read in.")
             .defaultValue("50")
             .propertyGroup(InstancePropertyGroup.PARTITION_SPLITTING).build();
+    UserDefinedInstanceProperty FIND_PARTITIONS_TO_SPLIT_TABLE_BATCH_SIZE = Index.propertyBuilder("sleeper.partition.splitting.finder.batch.size")
+            .description("The number of tables to perform partition splitting for in a single invocation. A separate " +
+                    "invocation of the lambda will be made for each batch when there are more tables than the batch " +
+                    "size.")
```

Comment:
We can clarify that it's batches for finding partitions to split, rather than to perform partition splitting.